### ISSUE TITLE
NPM Vulnerabilities 

### ISSUE THREAD

**Issue by dr3** [twolfson/css-validator#15]
**Describe the bug**
Package has a lot of vulnerabilities 

**To Reproduce**

1. Run `npm audit`
2. See error

```
found 47 vulnerabilities (15 low, 16 moderate, 15 high, 1 critical)
```

**Expected behavior**
Has no vulnerabilities 

**Additional context**
Just adding this issue as a reference, this package would be great to use in [our project](https://github.com/bbc/simorgh) but will need to fix vulnerabilities first. Hopefully I can find time to submit a PR :)

**Comment by twolfson** [comment]
A quick glance over the vulnerabilities seems to be that all of them are for our `devDependencies` which is why they haven't been prioritized. Happy to accept a PR for their upgrades though =)

**Comment by dr3** [comment]
Yeah, when you install the only prod vulnerability is moderate from `form-data` although it appears it wont be a clean version bump https://github.com/twolfson/css-validator/issues/10
